### PR TITLE
#192 added option for junit consolidated filename

### LINKED ISSUE #192: Junit add option for consolidate output filename
`var xmlFile = path.join(options.junit.path, 'TEST-' + testsuites.suite1.name.replace(/[^\w]/g, '') + '.xml');`

For the moment the above approach is intergrated.

Nice would be something like

options.junit.consolidatedFileName

### PR DESCRIPTION
**Pull request by PG2000**
fixes #192

### CODE REVIEW

**Comment by sindresorhus** [comment]
I don't see the point in adding a separate option. Just have `consolidate` take a string too which is the filename.

**Comment by webologen** [comment]
@sindresorhus Can you check why travis failed.

**Comment by sindresorhus** [comment]
@webologen You can. Just click the "Details" link. https://travis-ci.org/gruntjs/grunt-contrib-jasmine/builds/51554300

**Comment by PG2000** [comment]
@sindresorhus I can see an PhantomJS error. I dont think this is caused by my change.
What do you think ?

File: tasks/jasmine.js [hunk 1]
```diff
@@ -394,7 +394,7 @@ module.exports = function(grunt) {
     function writeJunitXml(testsuites){
       var template = grunt.file.read(options.junit.template || junitTemplate);
       if (options.junit.consolidate) {
-        var xmlFile = path.join(options.junit.path, 'TEST-' + testsuites.suite1.name.replace(/[^\w]/g, '') + '.xml');
+        var xmlFile = path.join(options.junit.path, options.junit.consolidate);
```
Comment (sindresorhus) [inline tasks/jasmine.js]: It should still support the previous behaviour when `true`.

File: docs/jasmine-options.md [hunk 2]
```diff
@@ -57,10 +57,11 @@ Default: undefined
 Path to output JUnit xml
 
 ## options.junit.consolidate
-Type: `Boolean`  
+Type: `Boolean,String`
```
Comment (sindresorhus) [inline docs/jasmine-options.md]:
```
Type: `Boolean`, `String` 
```

File: docs/jasmine-options.md [hunk 3]
```diff
@@ -57,10 +57,11 @@ Default: undefined
 Path to output JUnit xml
 
 ## options.junit.consolidate
-Type: `Boolean`  
+Type: `Boolean,String` 
 Default: `false`
 
-Consolidate the JUnit XML so that there is one file per top level suite.
+Consolidate the JUnit XML so that there is one file. If not false the string will be used as 
+the output filename.
```
Comment (sindresorhus) [inline docs/jasmine-options.md]: Instead:

> If a `string` it will be used as the output filename.

**Pull request closed without merging**